File: CliniCorp.Business/Models/Consulta.cs
namespace ProjetoDemo
{
    public class Consulta
    {
        public int Id { get; set; }
        public DateTime DataConsulta { get; set; }
        public string DescricaoConsulta { get; set; }
        public int Status { get; set; }
        public string StatusConsulta { get; set; }

        public Medico? Medico { get; set; }
        public Paciente? Paciente { get; set; }

    }
}

File: angular/src/app/consulta/models/consulta.ts
import { Medico } from "./medico";
import { Paciente } from "./paciente";

export interface Consulta {
    id: number,
    paciente: Paciente,
    medico: Medico,
    dataConsulta: Date,
    descricaoConsulta: string,
    status: number,
    statusConsulta: string,
    medicoId: number,
    pacienteId: number
}